### ISSUE TITLE
qubvel OpenCityChallenge UNet compatibility issues

### ISSUE THREAD

**Issue by chrischank** [chrischank/HOTOSM_OAM_codeV2#8]
Softmax deprecated in newer version of pytorch, made separate conda environment with downgraded torch and python, but not working so far :(
